extensions/prometheus-grafana-k8s/README.md

```diff
@@ -0,0 +1,98 @@
+# prometheus-grafana Extension
+
+
+This is the prometheus-grafana extension.  Add this extension to your json file as shown below to automatically enable prometheus and grafana in your new Kubernetes cluster.
+
+```
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes"
+    },
+    "masterProfile": {
+      "count": 1,
+      "dnsPrefix": "",
+      "vmSize": "Standard_DS2_v2",
+      "extensions": [
+        { 
+          "name": "prometheus-grafana-k8s"
+        }
+      ]
+    },
+    "agentPoolProfiles": [
+      {
+        "name": "agentpool1",
+        "count": 3,
+        "vmSize": "Standard_DS2_v2",
+        "availabilityProfile": "AvailabilitySet"
+      }
+    ],
+    "linuxProfile": {
+      "adminUsername": "azureuser",
+      "ssh": {
+        "publicKeys": [
+          {
+            "keyData": ""
+          }
+        ]
+      }
+    },
+    "extensionProfiles": [
+      { 
+        "name": "prometheus-grafana-k8s", 
+        "version": "v1",
+        "rootURL": "https://raw.githubusercontent.com/ritazh/acs-engine/feat-monitor/"
+      }
+    ],
+    "servicePrincipalProfile": {
+      "clientId": "",
+      "secret": ""
+    }
+  }
+}
+```
+
+
+The following script will be executed on the master:
+
+```
+$ prometheus-grafana-k8s.sh
+```
+
+You can validate that the extension is running as expected with the following commands:
+
+```
+$ kubectl get pods --show-all
+
+$ NAMESPACE=default

```

Comment:
Is this configurable? If not it probably should be.

Reply:
Currently it is not configurable. Definitely agreed that the end user should be able to specify a namespace.